experiments/pages/lightningModalLauncher/showScreenFlowModal.cmp

```diff
@@ -0,0 +1,35 @@
+<aura:component implements="flexipage:availableForAllPageTypes" access="global" >
+    <aura:attribute name="showModal" type="Boolean" default="false" access="private" />
+    <aura:attribute name="showCancel" type="Boolean" default="true" access="private" />
+    <aura:handler event="skuid:event" action="{!c.handleShowModalFlowEvent}" />
+
+    <aura:if isTrue="{!v.showModal}">
+        <section tabindex="-1" role="dialog" aria-modal="true" class="slds-modal slds-fade-in-open">
+            <div class="slds-modal__container">
+                <header class="slds-modal__header">
+                    <aura:if isTrue="{!v.showCancel}">
+                        <lightning:buttonIcon iconName="utility:close"
+                                              onclick="{! c.handleClose }"
+                                              alternativeText="close"
+                                              variant="bare-inverse"
+                                              class="slds-modal__close"/>
+                    </aura:if>
+                </header>
+                <div class="slds-modal__content">
+                    <lightning:flow aura:id="flow" onstatuschange="{!c.handleFlowStatusChange}" />
```

Comment:
PLease remove this <lightning: flow> section from the content div.    Replace it with some generic " Your component content goes here".  When I put the component in my org that didn't have any flow defined it balked at me.  The flow is not the point of this exercize.